frontend/src/bundles/common/pages/budgets/budget-details/budget-details.tsx

```diff
@@ -0,0 +1,256 @@
+            type: 'expense',
+        },
+    ],
+    currency: 'USD',
+};
+
+const BudgetDetails = (): JSX.Element => {
+    const doughnutData = [
+        {
+            total: 1150,
+            color: 'linear-gradient(95.5deg, #284B9F 0%, #102E68 100%)',
+        },
+        {
+            total: 1825,
+            color: 'linear-gradient(96.2deg, #FECC66 -30.03%, #F83062 95.13%)',
+        },
+        {
+            total: 1325,
+            color: 'linear-gradient(96.2deg, #FE66E6 -30.03%, #6933DD 95.13%)',
+        },
+    ];
+    const transactionData = [
+        {
+            id: '1',
+            category: 'Food',
+            name: 'faBagShopping',
+            date: '2022-03-23',
+            label: 'Supermarket',
+            amount: -35,
+            currency: '$',
+        },
+        {
+            id: '2',
+            category: 'Transport',
+            name: 'faCarAlt',
+            date: '2022-03-23',
+            label: 'Gas Station',
+            amount: -50,
+            currency: '$',
+        },
+        {
+            id: '3',
+            category: 'Shopping',
+            name: 'faStoreAltSlash',
+            date: '2022-04-22',
+            label: 'Clothing Store',
+            amount: 120,
+            currency: '$',
+        },
+        {
+            id: '4',
+            category: 'Food',
+            name: 'faBowlFood',
+            date: '2022-03-22',
+            label: 'Cafeteria',
+            amount: -10,
+            currency: '$',
+        },
+        {
+            id: '5',
+            category: 'Transport',
+            name: 'faCarAlt',
+            date: '2022-03-22',
+            label: 'Taxi Company',
+            amount: -25,
+            currency: '$',
+        },
+        {
+            id: '6',
+            category: 'Salary',
+            name: 'faMoneyBill',
+            date: '2023-03-30',
+            label: 'Electronics Store',
+            amount: 3500,
+            currency: '$',
+        },
+        {
+            id: '7',
+            category: 'Food',
+            name: 'faBowlFood',
+            date: '2024-03-21',
+            label: 'Restaurant',
+            amount: -60,
+            currency: '$',
+        },
+        {
+            id: '8',
+            category: 'Transport',
+            name: 'faCarAlt',
+            date: '2022-03-21',
+            label: 'Public Transport',
+            amount: -5,
+            currency: '$',
+        },
+        {
+            id: '9',
+            category: 'Salary',
+            name: 'faMoneyBill',
+            date: '2023-04-30',
+            label: 'Electronics Store',
+            amount: 3500,
+            currency: '$',
+        },
+    ];
+    const budget = getBudget;
+    const [active, setActive] = useState(false);
+
+    const handleCancel = useCallback(() => {
+        setActive(false);
+    }, []);
+    const handleModal = useCallback(() => {
+        setActive(true);
+    }, []);
+
+    // const { id } = useLocation();
```

Comment:
Does we need this?

Reply:
resolve in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/301/files#diff-7d399f4e833c64a50550a276c56abb5164f4a3ced886f28bdf20af73a21249b2](url)
Mikhaylo merged my branch and works with it